private/bufpkg/bufcheck/bufbreaking/internal/bufbreakingcheck/bufbreakingcheck.go

```diff
@@ -220,6 +239,33 @@ func isDeletedFieldAllowedWithRules(previousField bufprotosource.Field, message
 		(allowIfNameReserved && bufprotosource.NameInReservedNames(previousField.Name(), message.ReservedNames()...))
 }
 
+// CheckFieldSameCardinality is a check function.
+var CheckFieldSameCardinality = newFieldDescriptorPairCheckFunc(checkFieldSameCardinality)
+
+func checkFieldSameCardinality(
+	add addFunc,
+	_ *corpus,
+	_ bufprotosource.Field,
+	previousDescriptor protoreflect.FieldDescriptor,
+	field bufprotosource.Field,
+	descriptor protoreflect.FieldDescriptor,
+) error {
+	previousCardinality := getCardinality(previousDescriptor)
+	currentCardinality := getCardinality(descriptor)
+	if previousCardinality != currentCardinality {
+		// otherwise prints as hex
+		numberString := strconv.FormatInt(int64(field.Number()), 10)
+		// TODO: specific label location
```

Comment:
Addressing?

Reply:
Sorry, this was copied from existing label check and applies here as well. Do you want me to address it now? The label is actually always the first token of a field definition, so even without a `field.LabelLocation()`, this reports the correct location. Maybe I should just remove these TODOs?

Reply:
I've just removed the TODOs. I think it reports the correct location, even without a `field.LabelLocation()` accessor.